npc/cities/aldebaran.txt

```diff
@@ -1173,46 +1173,46 @@ aldeba_in,88,161,3	script	Kafra#05	4_F_KAFRA3,{
 
 		if (.@points) {
 			if (RESRVPTS < .@points) {
-				mes "[Kafra]";
-				mes "I'm sorry dear but you do not have enough points for this selection.";
+				mes("[Kafra]");
+				mes("I'm sorry dear but you do not have enough points for this selection.");
 				cutin "",255;
 				close;
 			}
 			RESRVPTS -= .@points;
-			callfunc "F_Lottery";
+			callfunc "F_Lottery");
```

Comment:
Only one bracket here.
You can change it to `callfunc("F_Lottery");` or leave it alone (as this PR is for Huld)

Reply:
Really sorry for this ! Will fix it ASAP.